CaloDiag/fcl/caloInspect.fcl

```diff
@@ -84,4 +81,4 @@ physics: {
 services.SeedService.baseSeed         :  99
 services.SeedService.maxUniqueEngines :  20
 
-physics.producers.CaloHitTruthMatch.caloShowerSimCollection  : "compressDigiMCs"
```

Comment:
AFAIK There is no CaloDigiMC object; what does CaloDigiTruthMatch represent?

Reply:
This refers to my inability to completely eradicate the CaloDigiTruthMatch name at the first try. This should be changed now.